### ISSUE TITLE
[Doc]: 针对淬火行动文档: 文档流程补充：多模态模型测试与精度测试与压测问题

### ISSUE THREAD

**Issue by wuzepei** [vllm-project/vllm-ascend#14636]
### 📚 The doc issue

1、视觉链路完全没有被压测
所有测试者执行的都是同一条命令：
vllm bench serve --dataset-name random --random-input 200 --num-prompts 200 --request-rate 1
其中 --dataset-name random 只会生成纯文本请求。Qwen3-VL 是视觉语言模型，但视觉编码器（Vision Tower）、图像预填充、多模态投影层从未在高负载下测试——只靠一条 curl 请求做了功能性验证。文档调优章节里明确列出的 “Multimodal (1080p)” 场景无人覆盖。
2、负载远未达到饱和
请求速率仅 1 RPS，峰值并发只有 12–14。因此报告中的输出吞吐 ~125 tok/s、总吞吐 ~322 tok/s 只是“巡航值”，并非这张卡的真实峰值能力。没有任何人做过高并发压测（如 --request-rate inf）。
3、没有精度验证
文档提供了精度评测方法（lm_eval 跑 mmmu_val、AISBench 跑 text_vqa），但几乎没人执行，部署正确性仅靠肉眼看一条 curl 返回结果。

### Suggest a potential alternative/fix

0. 前置：用优化配置重启服务（所有测试的基线）
#!/bin/bash
export HCCL_OP_EXPANSION_MODE="AIV"
export PYTORCH_NPU_ALLOC_CONF=expandable_segments:True
export OMP_PROC_BIND=false
export OMP_NUM_THREADS=1
export TASK_QUEUE_ENABLE=1
export ASCEND_RT_VISIBLE_DEVICES=0

MODEL=/workspace/Qwen3-VL-8B-Instruct

vllm serve $MODEL \
  --host 0.0.0.0 --port 8000 \
  --dtype bfloat16 \
  --served-model-name qwen3vl \
  --tensor-parallel-size 1 \
  --trust-remote-code \
  --max-num-seqs 128 \
  --max-model-len 32768 \
  --max-num-batched-tokens 16384 \
  --gpu-memory-utilization 0.91 \
  --async-scheduling \
  --compilation-config '{"cudagraph_mode": "FULL_DECODE_ONLY", "cudagraph_capture_sizes": [1,2,4,8,16,32]}' \
  --mm-processor-cache-gb 0

2、多模态测试
curl http://localhost:8000/v1/chat/completions \
  -H "Content-Type: application/json" \
  -d '{
    "model": "qwen3vl",
    "messages": [
      {"role": "user", "content": [
        {"type": "image_url", "image_url": {"url": "https://modelscope.oss-cn-beijing.aliyuncs.com/resource/qwen.png"}},
        {"type": "text", "text": "What is the text in the illustration?"}
      ]}
    ]
  }'

3、多模态负载压测（random-mm 合成数据）
vllm bench serve --model /workspace/Qwen3-VL-8B-Instruct \
  --dataset-name random-mm \
  --random-mm-base-items-per-request 1 \
  --random-mm-limit-mm-per-prompt '{"image": 1, "video": 0}' \
  --random-mm-bucket-config "{(1080, 1920, 1): 1.0}" \
  --enable-multimodal-chat \
  --random-input 200 --num-prompts 200 --request-rate 1 \
  --save-result --result-dir ./results/

4、精度测试 lm_eval 离线评测
pip install lm_eval

lm_eval \
    --model vllm-vlm \
    --model_args pretrained=/workspace/Qwen3-VL-8B-Instruct,max_model_len=8192,gpu_memory_utilization=0.7 \
    --tasks mmmu_val \
    --batch_size 32 \
    --apply_chat_template \
    --trust_remote_code \
    --output_path ./results/lm_eval

**Comment by Annarine** [comment]
@jyoung6652 The visual pipeline has never been stress-tested, the load never reached saturation, and no accuracy verification was performed.
